zitadel/helper/client.go

```diff
@@ -91,7 +91,7 @@ func GetAdminClient(ctx context.Context, info *ClientInfo) (*admin.Client, error
 		if adminClient == nil {
 			client, err := admin.NewClient(ctx,
```

Comment:
The context provided until now is cancelled in the planning or applying, the logic changed in the zitadel-go unfortunately.

Reply:
I understand that, that's why we pass context.Background. You just missed to do that for the admin client, correct?

BTW, is it not cleaner if we only create one new context with context.Background() (and also with its own timeout) and pass that to all clients instead of passing a new context to each client?

Reply:
removed the unnecessary changes, the problem was simple that the ctx used for the TokenSource was cancelled